packages/core/__tests__/login.test.ts

```diff
@@ -11,7 +11,7 @@ describe('login', () => {
       writable: true,
       configurable: true
     });
-    await client.login('token');
+    await client.login('token'); // TODO: bug with rewrite local config file
```

Comment:
Can this todo be fixed?

Reply:
I'd like to left it as a tech debt for now.